packages/node-opentelemetry/src/otel.ts

```diff
@@ -23,6 +24,7 @@ type SDKConfig = {
   betaMode?: boolean;
   consoleCapture?: boolean;
   instrumentations?: InstrumentationConfigMap;
+  additionalInstrumentations?: InstrumentationBase[];
```

Comment:
Can you also add this to the README.md ?

Reply:
Sure thing!

Reply:
Let me know how that looks or if you want me to write more context on the field.

Reply:
I think we also want to add a new section under '(Optional) Advanced Instrumentation Configuration' to tell people how to add 3rd-party instrumentation pkgs and put one example there. Also would be nice to add a list of current auto instrumentation pkgs (https://www.npmjs.com/package/@opentelemetry/auto-instrumentations-node#supported-instrumentations)